### ISSUE TITLE
Toolbar causes an error when using `kendo-toolbar-dropdownbutton` with ngIf/ngFor

### ISSUE THREAD

**Issue by destus90** [telerik/kendo-angular#4465]
### Description

When you have a tool under ngIf directive, toolbar will throw an error in the browser's console.

### Steps To Reproduce

1. Go to https://stackblitz.com/edit/angular-jv9hm3-24qd4a?file=src%2Fapp%2Fapp.component.ts
2. Check the browser's console.

### Actual Behavior

Error is thrown

### Expected Behavior

No error

### Browser

Chrome

### Browser version

latest

### OS type

Windows

**Comment by mbechev** [comment]
The reported error is also reproduced when using **ngFor**:

```html
<kendo-toolbar>
      <kendo-toolbar-dropdownbutton *ngFor="let item of items" [text]="item.text" [data]="data">
      </kendo-toolbar-dropdownbutton>
</kendo-toolbar>
```

https://stackblitz.com/edit/angular-532tbn-mfz3hx

**Comment by Raisolution** [comment]
Fix available in `v17.1.1`
